client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/TicketTypeUpdateTable.tsx

```diff
@@ -95,10 +111,18 @@ export const TicketTypeUpdateTable = (props: TicketTypeTableProps) => {
               {getTicketTypePrice(1, 'description', ticketTypes)}
             </td>
             <td className={'px-2 border border-white'}>
-              {getTicketTypePrice(1, 'price', ticketTypes)}
+              <input
+                type='text'
+                value={ticketPrice}
+                onChange={(e) => handlePriceChange(e.target.value)}
+                />
             </td>
             <td className={'px-2 border border-white'}>
-              {getTicketTypePrice(1, 'concessions', ticketTypes)}
+              <input
+                type='text'
+                value={concessionPrices}
+                onChange={(e) => handleConcessionPrices(e.target.value)}
+                />
```

Comment:
For both of these inputs, could you add a `className` with `w-[75px]` and `bg-gray-100` so that the input blends in more and takes up a bit less horizontal space

Reply:
Changed. 
![image](https://github.com/WonderTix/WonderTix/assets/97830813/d5d3e528-68cc-4379-a37a-8aa7b5f90eda)